features/password_reset.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Password Reset Operations
+  In order to allow users to reset their passwords
+  As a system
+  I want to provide password reset functionality
+
+  Scenario: Requesting password reset for existing user
+    Given user with email "reset@test.com" exists
+    And requesting password reset for email "reset@test.com"
+    When POST request is send to "/api/users/{id}/reset-password"
```

Comment:
We should remove the id from the url path, because we don't need to know anything for this feature except the email, after changing the api platform configuration and all related code, please rewrite this behat tests